gap/congruences/congpairs.gi

```diff
@@ -442,8 +477,27 @@ function(cong1, cong2)
 end);
 
 InstallMethod(\in,
-"for dense list and finite semigroup congruence by generating pairs rep",
-[IsDenseList, IsFiniteCongruenceByGeneratingPairsRep],
+"for dense list and enumerable semigroup congruence",
+[IsDenseList, IsEnumerableSemigroupCongruence],
+function(pair, cong)
+  local S;
+  S := Range(cong);
+  if Size(pair) <> 2 then
+    ErrorNoReturn("Semigroups: \\in (for a congruence): usage,\n",
+                  "the first arg <pair> must be a list of length 2,");
+  elif not (pair[1] in S and pair[2] in S) then
+    ErrorNoReturn("Semigroups: \\in (for a congruence): usage,\n",
```

Comment:
I'm not sure (genuinely not sure) what the right behaviour of `\in` should be. Here, if the pair of elements aren't both in the semigroup, then obviously the pair isn't in the congruence. But should this trigger an error, or return false?

Compare with the following:

```
gap> S := FullTransformationMonoid(1);;
gap> EmptyPartialPerm() in S;
false
```
The empty partial perm is a partial perm, and `S` is a transformation semigroup, so obviously it's not in `S`. But this returns `false` rather than an error.

It's not something I care deeply about, but I think it's something worth thinking about, especially for the sake of consistency with the rest of the package and GAP.